pages/_document.jsx

```diff
@@ -33,6 +33,7 @@ export default class MyDocument extends Document {
       <Html>
         <Head>
           {this.props.styleTags}
+          <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Source+Sans+Pro:wght@400;600;700&display=swap" />
```

Comment:
Google fonts suggests adding 
```
<link rel="preconnect" href="https://fonts.googleapis.com">
<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
```

Ant it would be better to load fonts before css, so put it above `{this.props.styleTags}`